python/tvm/relay/frontend/pytorch.py

```diff
@@ -1757,6 +1759,20 @@ def _impl(inputs, input_types):
     return _impl
 
 
+def _stack(prelude):
+    def _impl(inputs, input_types):
+        if isinstance(inputs[0], list):
+            # a static python list of tensors
+            dim = inputs[1]
+            return _op.stack(inputs[0], dim)
+        else:
+            # List ADT case
+            # TODO: is there a better way to check if an input is a List ADT?
```

Comment:
let's put an assignee after TODO

Reply:
It seems if I do `infer_type` here, I get 
```
List[Tensor[(8, 8), float32]]
```

Also, if I do `prelude.mod.get_global_type_var("List")`, I get
```
List
```

so I'm looking for a way to test a type equality, any idea? cc @kevinthesun

Reply:
it seems this works

```
ty = _infer_type_with_prelude(inputs[0], prelude)
list_ty = prelude.mod.get_global_type_var("List")
assert ty.func == list_ty
            
```

Reply:
A proper assertion added and TODO removed